janus_core/calculations/geom_opt.py

```diff
@@ -289,15 +296,30 @@ def run(self) -> None:
 
         converged = self.dyn.run(fmax=self.fmax, steps=self.steps)
 
-        s_grp = spacegroup(self.struct, self.symmetry_tolerance, self.angle_tolerance)
-        self.struct.info["final_spacegroup"] = s_grp
-
         # Calculate current maximum force
         if self.filter_func is not None:
             max_force = linalg.norm(self.filtered_struct.get_forces(), axis=1).max()
         else:
             max_force = linalg.norm(self.struct.get_forces(), axis=1).max()
 
+        if self.symmetrize:
+            snap_symmetry(self.struct, self.symmetry_tolerance)
```

Comment:
probably you want also the angle passed in here. also should it be snap_to_symmetry?

Reply:
`refine_symmetry` only takes one parameter for identifying the symmetry, so how would the angle be used? https://wiki.fysik.dtu.dk/ase/_modules/ase/spacegroup/symmetrize.html#refine_symmetry

Reply:
yes the issue is upstream, they use it inconsitent
they do not pass angle_tolerance even if spglib needs it, I would suggest correcting this upstream. there are 2 or 3 places where needs to be corrected, practically api of refine_symmetry and maybe 2 functions.

Reply:
In spglib itself, it does say 

```
    angle_tolerance : float
        Symmetry search tolerance in the unit of angle deg.
        Normally it is not recommended to use this argument.
        See a bit more detail at :ref:`variables_angle_tolerance`.
        If the value is negative, an internally optimized routine is used to
        judge symmetry.
```

Either way, even if we contributed to the upstream now, we wouldn't be able to consistently use it until the next tagged release.

Reply:
yes will not be now but will be good to be done. I know the warning issue is that we are inconsitent